### PR TITLE
Updated Documentation

### PR DESCRIPTION
**Pull request by arayofcode**
Added instructions to install on Arch Linux and to fix the `No File or Directory` error. It would be great if you could review this and accept if this is helpful

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,9 +14,10 @@ It has been developed and released by the [Amnesty International Security Lab](h
 
 ## Installation
 
-First you need to install dependencies, on Linux `sudo apt install python3 python3-pip libusb-1.0-0` or on MacOS `brew install python3 libusb`.
+First you need to install dependencies, on Linux `sudo apt install python3 python3-pip libusb-1.0-0` or on MacOS `brew install python3 libusb`. For Arch Linux, `sudo pacman -S python-pip libusb`. 
+
+Then you can install mvt from pypi with `pip install mvt` (use `--user` in case of Arch Linux to avoid conflicts with pacman packages), or directly from sources:
```
Comment (NicolasCARPi) [inline README.md:19]: The `--user` flag is actually a good thing everywhere IMHO ;)

File: docs/install.md [hunk 2]
```diff
@@ -12,6 +12,15 @@ sudo apt install python3 python3-pip libusb-1.0-0
 
 *libusb-1.0-0* is not required if you intend to only use `mvt-ios` and not `mvt-android`.
 
+### Note: For Arch Linux
+
+Use the following command for installing the tools:
+
+```bash
+sudo pacman -Syyy
```
Comment (NicolasCARPi) [inline docs/install.md:20]: One `y` is enough, two is overkill, but three? I don't think it has any effect.

Comment (arayofcode) [reply docs/install.md:20]: I see. My bad, I'll correct it to one y. Thank you!

**Comment by NicolasCARPi** [comment]
@arayofcode you know you could have just pushed new commits to your branch, no need to close it and make a new one ;)

**Comment by arayofcode** [comment]
@NicolasCARPi This is the first time I have ever contributed to any project. My bad again :p I'll keep this in mind the next time. Thank you so much for your help!

**Comment by arayofcode** [comment]
@NicolasCARPi Made a new pull request at https://github.com/mvt-project/mvt/pull/56. Can you please check and comment on what I can improve on?

**Pull request closed without merging**